src/main/java/seedu/address/model/internship/Requirement.java

```diff
@@ -1,32 +1,84 @@
 package seedu.address.model.internship;
 
-import seedu.address.model.wrapper.NonEmptyString;
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
 
 /**
  * Requirement is an optional field in an InternshipItem.
  * Guarantees: immutable; is valid as declared in {@link #isValidRequirement(String)}
  */
-public class Requirement extends NonEmptyString {
+public class Requirement {
 
-    public static final String MESSAGE_CONSTRAINTS = "Requirements should not be blank";
+
+    public static final String MESSAGE_CONSTRAINTS = "Requirement have a maximum of 2 words and not be blank";
```

Comment:
I think i missed why is it limited to 2 words in particular?

Reply:
We were discussing this previous time and we decided that this allows an easier skill matching. 

Because the user can key in any type of requirements that will render the skill matching useless.